flux_exchange.F90

```diff
@@ -1365,6 +1365,16 @@ subroutine flux_exchange_init ( Time, Atm, Land, Ice, Ocean, Ocean_state,&
     allocate( ice_ocean_boundary%calving_hflx  (is:ie,js:je) ) ;    ice_ocean_boundary%calving_hflx = 0.0
     allocate( ice_ocean_boundary%p        (is:ie,js:je) ) ;         ice_ocean_boundary%p = 0.0
     allocate( ice_ocean_boundary%mi       (is:ie,js:je) ) ;         ice_ocean_boundary%mi = 0.0
+    
+    if (Ice%Ice_state%do_icebergs) then
```

Comment:
This requires a change in SIS public interface. That is fine, but SIS does not have Ice%Ice_state. Moreover SIS2 comments say that Ice%Ice_state is "private to SIS2 and is not to be shared with other components and modules". Ice% on the other hand is meant to be public in both SIS and SIS2. Could you modify SIS2 and icebergs so that we could use  Ice%do_icebergs here rather than Ice%Ice_state%do_icebergs ?